.github/CODEOWNERS

```diff
@@ -239,7 +239,6 @@ packages/react-components/react-toolbar @microsoft/teams-prg @chpalac @ling1726
 packages/react-components/react-toolbar/library @microsoft/teams-prg @chpalac @ling1726
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31642&commitId=1757549cc9ea58fe27e75affd2614407f11d4353&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|0| Added|
|Avatar Converged.basic.normal.chromium.png|0| Added|
|Avatar Converged.badgeMask.normal.chromium.png|0| Removed|
|Avatar Converged.size+image+badge.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Shape.default.chromium.png|0| Added|
|SwatchPicker Converged.Spacing.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>